projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-method.component.spec.ts

```diff
@@ -117,19 +86,18 @@ describe('PaymentMethodComponent', () => {
   });
 
   it('should call ngOnInit to get existing payment methods if they do not exist', () => {
-    mockUserSelectors.getPaymentMethods.next([]);
+    mockCheckoutService.paymentMethods$.next([]);
     component.ngOnInit();
     component.existingPaymentMethods$.subscribe(() => {
-      expect(service.loadUserPaymentMethods).toHaveBeenCalled();
+      expect(mockCheckoutService.loadUserPaymentMethods).toHaveBeenCalled();
     });
   });
 
   it('should call ngOnInit to get existing payment methods if they exist', () => {
-    const mockPayments = mockPaymentMethods;
-    mockUserSelectors.getPaymentMethods.next(mockPayments);
+    mockCheckoutService.paymentMethods$.next(mockPaymentMethods);
     component.ngOnInit();
     component.existingPaymentMethods$.subscribe(data => {
-      expect(data).toBe(mockPayments);
+      expect(data).toBe(mockPaymentMethods);
```

Comment:
Please either use `done()` or assign the result to a variable defined outside

Reply:
for tests in checkout, they will be refactored in checkout facade ticket #154